sound/soc/sof/ipc4-topology.c

```diff
@@ -1650,12 +1655,31 @@ static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
 		control_data = scontrol->ipc_control_data;
 		if (control_data->data->blob_type != SOF_IPC4_MOD_INIT_INSTANCE)
 			continue;
-
+		data_size = control_data->data->size;
```

Comment:
What if data_size = 0? Or there is no SOF_IPC4_MOD_INIT_INSTANCE blob type in the kcontrol?